Makefile

```diff
@@ -158,8 +158,6 @@ generate-contour-crds:
 generate-gateway-crds:
 	@echo "Generating Gateway API CRD YAML documents..."
 	@kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=${GATEWAY_API_VERSION}" > config/crd/gateway/01-crds.yaml
-	@echo "---" >> config/crd/gateway/01-crds.yaml
-	@curl -s https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/${GATEWAY_API_VERSION}/config/crd/experimental/gateway.networking.k8s.io_referencepolicies.yaml >> config/crd/gateway/01-crds.yaml
```

Comment:
They ended up adding back ReferencePolicy to the main CRD manifest a/o v0.5.0 so this extra pull is no longer needed.